core/controllers/topic_viewer.py

```diff
@@ -61,7 +61,11 @@ class TopicPageDataHandler(base.BaseHandler[Dict[str, str], Dict[str, str]]):
 
     @acl_decorators.can_access_topic_viewer_page
     def get(self, topic_name: str) -> None:
-        """Handles GET requests."""
+        """Access a topic viewer page.
```

Comment:
Access --> Accesses (let's keep tenses consistent)

Reply:
I have updated the docstrings.